### PR TITLE
docs(multitenancy): Add Warning about scheduledRefreshContexts

### PR DESCRIPTION
**Pull request by ovr**
Hello!

refs https://github.com/cube-js/cube.js/issues/1904

Thanks

### CODE REVIEW

File: docs/content/Configuration/Multitenancy-Setup.md [hunk 1]
```diff
@@ -20,6 +20,13 @@ The options are:
 - `preAggregationsSchema`
 - `queryTransformer`
 
+<!-- prettier-ignore-start -->
+[[warning |]]
+| In the [scheduled refresh](caching#pre-aggregations-background-refresh) leaving [`scheduledRefreshContexts`](config#options-reference-scheduled-refresh-contexts)  
+| unconfigured will lead to issues where the security context will be `undefined`. This is because there is no way for
+| Cube.js to know how to generate a context without the required input.
+<!-- prettier-ignore-end -->
+
```
Comment (hassankhan) [inline docs/content/Configuration/Multitenancy-Setup.md:29]: Not sure about the placement of this - I think we should create a section in this (Multitenancy) page specifically for scheduled refreshes, WDYT @keydunov @ovr?

File: docs/content/Configuration/Config.md [hunk 2]
```diff
@@ -371,13 +371,22 @@ module.exports = {
...
+
+Best practice is to run `scheduledRefreshTimer` in a separate worker Cube.js
+instance. For serverless deployments [REST API](rest-api#api-reference-v-1-run-scheduled-refresh) should be used
+instead of timer.
+
+There are some options that are important for the scheduled refresh:
+
+- [`scheduledRefreshTimeZones`](#options-reference-scheduled-refresh-time-zones)
+- [`scheduledRefreshContexts`](#options-reference-scheduled-refresh-contexts)
+
```
Comment (hassankhan) [inline docs/content/Configuration/Config.md:384]:
```suggestion
You may also need to configure [`scheduledRefreshTimeZones`](#options-reference-scheduled-refresh-time-zones) and [`scheduledRefreshContexts`](#options-reference-scheduled-refresh-contexts).
```

**Review by hassankhan — CHANGES_REQUESTED** [review]
Minor changes 😀 The only "problem" is the edits to Multitenancy - I agree we need to update the docs, but I think it should be more than a warning.

File: docs/content/Configuration/Multitenancy-Setup.md (hunk 1 above)
Comment (ovr) [reply docs/content/Configuration/Multitenancy-Setup.md:29]: We have the issue: https://github.com/cube-js/cube.js/issues/1904
Where users don't know about that they need to use `scheduledRefreshContexts` with multitenancy. It's why I added a reference as warning block to indicate that it's important.

Comment (hassankhan) [reply docs/content/Configuration/Multitenancy-Setup.md:29]: @ovr I totally agree we should keep the warning 😄 I just meant maybe we should also make a 'Scheduled Refresh' section on this page and then move the warning there. The current placement of the warning seems a bit disconnected from the rest of the text around it.

**Review by hassankhan — APPROVED** [review]

**Pull request merged**